### PR TITLE
Add profile page

### PR DESCRIPTION
**Pull request by konstrike**
Add profile page

### CODE REVIEW

File: frontend/src/app/navbar/userNav.jsx [hunk 1]
```diff
@@ -16,9 +21,14 @@ const UserNav = (): React.Node => {
     return (
         <>
             <Nav.Item className="userName">
-                <Nav.Link eventKey="disabled" disabled>
+                <NavLink
+                    className="nav-link"
+                    isActive={() => location.pathname === '/profile'}
+                    to={'/profile'}
```
Comment (teodorvoicencu) [inline frontend/src/app/navbar/userNav.jsx]: Move `'/profile'` inside a const variable. Call it route or idk

File: frontend/src/app/pages/desktop.userProfilePage.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+                        <Form.Group as={Row} className="ml-3">
+                            <Form.Label column sm="2">
+                                Email
+                            </Form.Label>
+                            <Col sm="10">
+                                <Form.Control plaintext readOnly defaultValue={email} />
+                            </Col>
+                        </Form.Group>
+                    </Form>
```
Comment (teodorvoicencu) [inline frontend/src/app/pages/desktop.userProfilePage.jsx]: Don't use form!!!

File: frontend/src/app/pages/desktop.userProfilePage.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import { useSelector } from 'react-redux';
+
+import './styles/desktop.userProfilePage.scss';
+
+const UserProfile = (): React.Node => {
+    const user = useSelector(state => state.user.name);
+    const email = useSelector(state => state.user.username);
+    return (
+        <div className="row fix justify-content-center mt-3 w-100">
+            <img src="https://st4.depositphotos.com/1012074/20946/v/450/depositphotos_209469984-stock-illustration-flat-isolated-vector-illustration-icon.jpg" />
```
Comment (umihnea) [inline frontend/src/app/pages/desktop.userProfilePage.jsx]: Please extract this sausage URL in a constant above the component. Like `const defaultImageSource = ...` or `imagePath` or `imageUrl`. For the future, maybe find a way to store files.

**Review by umihnea — COMMENTED** [review]
Please run `yarn fix` and rebase with master.

File: frontend/src/app/pages/desktop.userProfilePage.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            <div className="row fullWidth justify-content-center mt-3">
+                <div className="box">
+                    <div className="card-header text-center">
+                        <h2>My Profile</h2>
```
Comment (umihnea) [inline frontend/src/app/pages/desktop.userProfilePage.jsx:19]: Can you make this more generic like "X's Profile" depending on the username?

Comment (konstrike) [reply frontend/src/app/pages/desktop.userProfilePage.jsx:19]: Done

**Review by umihnea — APPROVED** [review]
You're weak, Sasuke. You don't have enough... _React_.

**Pull request merged**